Project_Proposal.Rmd

```diff
@@ -0,0 +1,60 @@
+
+# Problem / Question
+
+How has access to community gardens and fridges improved food security in Buffalo, NY and surrounding areas?
+
+# Inspiring Examples
+
+## Example 1
+
+![Buffalo Go Green is an organization in Buffalo that has a community garden in the East Side of Buffalo, and works with other organizations across Buffalo to provide food security to residents](https://i0.wp.com/buffalogogreen.org/wp-content/uploads/2022/04/UrbanFarm.jpg?w=500&ssl=1)
+
+Buffalo Go Green is an organization in Buffalo that has a community garden in the East Side of Buffalo, and works with other organizations across Buffalo to provide food security to residents
+
+## Example 2
+
+![This is an example of a map made by researchers at UB that shows food resources in Buffalo. This is along the lines of what I would like to produce with the additional feature of mapping food security](https://www.buffalorising.com/wp-content/uploads/2020/04/Food-Map-Buffalo-NY-730x419.png)
+This is an example of a map made by researchers at UB that shows food resources in Buffalo. This is along the lines of what I would like to produce with the additional feature of mapping food security
+
+
+## Example 3
+
+![This is an interactive map created by Feeding America where you can see levels of food insecurity in each state and county. This is a map of the united states. I was able to get the data for Erie County from this organization, and will be using their data for food insecurity.](https://blog.richmond.edu/livesofmaps/files/2017/02/Feeding-America-Map-768x613.png)
+This is an interactive map created by Feeding America where you can see levels of food insecurity in each state and county. This is a map of the united states. I was able to get the data for Erie County from this organization, and will be using their data for food insecurity.
+
+
+# Proposed data sources
+
+I was able to gather two data sets from Feeding America on food insecurity. They are CSV files, and list data for food security in Erie County for all ages and then also for senior citizens. This data is from 2009-2020. I also found a CSV file that lists community fridges in Buffalo and surrounding areas. I am expecting to get a data set from Buffalo food lab on community gardens in Buffalo. I have contacted them in regards to using their data set, but have not yet heard anything. I will try emailing them again, and if that doesn't work. I can use their data on their website of community gardens and create my own csv file with a list of community gardens.

```

Comment:
are these spatial datasets?  Or just single values for the whole county.  I imagine this varies tremendously over space.  

Also, what other factors might be useful here?  Demographic data (population, income, etc.) from the census?